src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -84,6 +85,10 @@ public void displayResult(CommandResult result) {
         if(resultPersons.isPresent()) {
             display(resultPersons.get());
         }
+        final Optional<ArrayList<String>> resultCommands = result.getRelevantCommands();
+        if(resultCommands.isPresent()) {
```

Comment:
i followed the way they did it for the earlier if statement. if you want me to change it i can